bands/templates/bands/bio.html

```diff
@@ -9,7 +9,7 @@
 <br/> <br/>
 
 <form method="post" action="bio" class="form-horizontal" role="form" enctype="multipart/form-data">
-    <input type="hidden" name="id" value="{{ band.id }}" />
+    <input type="hidden" name="band_id" value="{{ band.band_id }}" />
```

Comment:
One thing to note because it's better to be deliberate about these kinds of security considerations: this approach allows anyone who knows the band id to edit the info for a band.  In other words, if one band goes to their checklist and then opens up this form, they could edit the foreign key here to fill out the info for a different band.

I'm totally okay with this here, because currently if you know the band id, you can already just visit the band checklist for that band anyway, so this doesn't make us any less secure than we already were :)

However, I wanted to mention this because in the MIVS plugin, we actually added some extra security around this to protect against someone who knew the game id but not the studio id being able to modify the game.  This is actually a concern for the MIVS plugin because there are some URLs where the game id is exposed but the studio id is not.  If this ever becomes true for the band, we may need to start adding additional session-based protections for this stuff.